package-lock.json

```diff
@@ -14,6 +14,9 @@
         "packages/govuk-frontend-review",
         "docs/examples/*"
       ],
+      "dependencies": {
+        "@types/marked": "*"
```

Comment:
Sorry @domoscargin can you run `npm install` again?

It's left a fragment of a previous change in the **package-lock.json**

Installing again should do a tidy up

Reply:
Done!